Scripts/Runtime/Util/Random/RandomExtensions.cs

```diff
@@ -0,0 +1,38 @@
+using System.Runtime.CompilerServices;
+using Unity.Jobs.LowLevel.Unsafe;
+using Unity.Mathematics;
+
+namespace Anvil.Unity.DOTS.Util
+{
+    /// <summary>
+    /// Extension methods for use with a <see cref="Random"/> instance.
+    /// </summary>
+    public static class RandomExtensions
+    {
+        private const uint SPREAD = uint.MaxValue / JobsUtility.MaxJobThreadCount;
+        
+        /// <summary>
+        /// Given an instance of <see cref="Random"/>, this extension modifies the <see cref="Random"/>'s state so that
+        /// multiple threads using the same initial state will have output different random values based on the thread
+        /// index.
+        /// </summary>
```

Comment:
```suggestion
        /// <summary>
        /// Given an instance of <see cref="Random"/>, this extension modifies the <see cref="Random"/>'s state so that
        /// multiple threads using the same initial state will output different random values based on the thread
        /// index.
        /// </summary>
```

Reply:
Fixed